docs/UserGuide.md

```diff
@@ -14,6 +14,7 @@ ATAS is a **desktop app for managing students’ particulars, optimized for use
   * [Listing all students](#list)
   * [Locating students by name](#find)
   * [Deleting a student](#delete)
+  * [Edit a student](#edit)
```

Comment:
Shouldn't this be renamed to `Editing a student` to fit the rest of the item naming?